.github/workflows/main.yaml

```diff
@@ -189,6 +219,58 @@ jobs:
           type: ondevice
           os: linux
 
+  # Runs on-host integration and unit tests.
+  on-device-test:
+    needs: [initialize, build]
+    if: needs.initialize.outputs.on_device_test == 'true' && (contains(github.workflow, '_nightly') || contains(github.event.pull_request.labels.*.name, 'on_device'))
```

Comment:
ditto (no nightly workflows).

Reply:
What about postsubmit (push event), why are we not handling it?

Reply:
Resolved